Content/ReleaseBriefings/4_6/Section_NewFeatures.tex

```diff
@@ -46,5 +46,67 @@
 
 %==========================================================================
 
+\begin{frame}[fragile]\label{sec:new_features}
+
+  {\Huge Kokkos::Graph features}
+
+  \vspace{10pt}
+
+\end{frame}
+
+\begin{frame}[fragile]{Kokkos::Graph recap}
+ \begin{itemize}
+     \item Allows creating a graph of kernels
+     \item Executed in order when calling \texttt{submit(ExecutionSpace)}
+      \begin{code}[keywords={auto}]
+    ...
+    auto graph = Kokkos::Experimental::create_graph(exec, [&](auto root) {
+      auto node_A = root.then_parallel_for(
+                    policy_t(exec, 0, 1),
+                    FetchValuesAndContribute(data, index_A, value_A));
+      auto node_B = root.then_parallel_for(
+                    policy_t(exec, 0, 1),
+                    FetchValuesAndContribute(data, index_B, value_B));}
+    ...
+    graph.submit(exec);
```

Comment:
Please make it more appealing, and aligned with the example I gave in https://github.com/kokkos/kokkos-core-wiki/pull/648. And remove `exec` because it may be confusing and isn't serving any particular purpose in this case.
```c++
    auto graph = Kokkos::create_graph([&](auto root) {
        auto node_A = root.then_parallel_for("workload A", ...policy..., ...functor...);

        auto node_B = node_A.then_parallel_for("workload B", ...policy..., ...functor...);
        auto node_C = node_A.then_parallel_for("workload C", ...policy..., ...functor...);

        auto node_D = Kokkos::when_all(node_B, node_C).then_parallel_for("workload D", ...policy..., ...functor...);
    });

    graph.instantiate();

    graph.submit()
```

Reply:
done in https://github.com/kokkos/kokkos-tutorials/pull/125/commits/ba8f0b308d21e5508c5a7d60179eaa58717f799d